scripts/sanitycheck

```diff
@@ -176,11 +176,17 @@ from collections import OrderedDict
 from itertools import islice
 import yaml
 
+import logging
+log_format = "%(levelname)s %(name)s::%(module)s.%(funcName)s():%(lineno)d: %(message)s"
```

Comment:
how is this used? i don't see any modifications for the verbose() or error() functions. move this to scl.py?